.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ on:
   pull_request:
 
 env:
-  docker_repository: docker.synapse.org/syn25992884/bert-phi-annotator
+  docker_repository: docker.synapse.org/syn22277123/phi-annotator-huggingface
```

Comment:
If I'm understanding your plan, shouldn't there actually be no particular Docker image named `phi-annotator-huggingface`? E.g. they should have come with some suffix like `phi-annotator-huggingface-bert-large`

Reply:
Correct. Once we know their name, we can place them all in `docker-compose.yml` and comment out all but one.